### ISSUE TITLE
Release 9.2.1

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#3255]
Shall we get https://github.com/stylelint/stylelint/pull/3250 merged and then release a patch? This should partially address https://github.com/stylelint/stylelint/issues/3248 and https://github.com/stylelint/stylelint/pull/3250#issuecomment-378606000.

The outcome of https://github.com/stylelint/stylelint/issues/3252 seems likely to introduce a breaking change. So, I don't believe there's harm updating `postcss-html` for the 9.x branch.

- [x] stylelint release
- [x] stylelint-config-recommended update/release
- [x] stylelint-config-standard update/release
- [x] stylelint-demo update
- [x] stylelint.io update
- [x] tweet

cc @stylelint/core

**Comment by jeddy3** [comment]
This is now dependent on PR https://github.com/stylelint/stylelint/pull/3261.

Once that is merged, it'll close #3252. I'll create another issue to further discuss processors and syntax, and what we intend to do about them going forward.

**Comment by jeddy3** [comment]
🎉 Done.

I updated the website, but it's not showing up yet. Perhaps a Cloudflare caching issue?

---

I'm going to try to triage a couple of issues before I go on holiday this Friday. I'll mostly be offline for the next couple of weeks.

**Comment by ntwb** [comment]
Awesome, thanks @jeddy3, enjoy your holiday.

I hope to have some more time next week so I'll be able to help out some more
